src/single.ts

```diff
@@ -121,6 +122,10 @@ export class RegionRatelimit extends Ratelimit<RegionContext> {
      * The duration in which `tokens` requests are allowed.
      */
     window: Duration,
+    /**
+     * How many custom rates(if any) are allowed per window.
+     */
+    rates?: number,
```

Comment:
I didn't quite understand the necessity for the `rates` argument here. When defining a window, the total token count can take any value, independent of the consume rate. So I can use `fixedWindow(400, '10 s')` and consume 400 tokens by a rate of 10. With this signature, the usage is `fixedWindow(10, "10 s", 400)` which is quite hard to understand. What would be the purpose of providing `10` as first argument? I think we should keep the api simple

Reply:
It will all come around and everything will make sense. Just give me sometime.

Reply:
No worries, I thought this was the proposal 👍🏻